modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java

```diff
@@ -75,13 +79,13 @@
 /**
  * The IAM Identity Service API allows for the management of Account Settings and Identities (Service IDs, ApiKeys).
  *
- * @version v1
+ * API Version: 1.0.0
  */
 public class IamIdentity extends BaseService {
 
   public static final String DEFAULT_SERVICE_NAME = "iam_identity";
 
-  public static final String DEFAULT_SERVICE_URL = "https://iam.cloud.ibm.com";
+  public static final String DEFAULT_SERVICE_URL = "https://iam.test.cloud.ibm.com";
 
```

Comment:
The default url here should be the production IAM endpoint URL.  Please make sure you are using the production version of the API definition when re-generating SDK code.

Reply:
yes correct, I will update the PR now.. :)